lib/matplotlib/colorbar.py

```diff
@@ -180,6 +180,9 @@
 to which the colorbar is attached; but it is a manual method requiring
 some trial and error. If the colorbar is too tall (or a horizontal
 colorbar is too wide) use a smaller value of *shrink*.
+Further, *shrink* and *cax* kwargs are incompatible with each other and
```

Comment:
"Further, *shrink* " -> "The *shrink*".   Take or leave this, but "further" is kind of awkward unless really needed...

Reply:
I'd actually just replace this whole paragaph with: "Note that if *cax* is specified it determines the size of the colorbar and *shrink* and *aspect* kwargs are ignored"

If you are going to point this out for *shrink* it should also be pointed out for *aspect*.

Reply:
Replaced with "The *shrink* kwarg provides a simple way to scale the colorbar with respect
to the axes. Note that if *cax* is specified it determines the size of the
colorbar and *shrink* and *aspect* kwargs are ignored."